projects/plugins/boost/app/assets/src/js/features/speculation-rules/speculation-method.tsx

```diff
@@ -0,0 +1,68 @@
+import { useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { z } from 'zod';
+import ModuleSubsection from '$features/ui/module-subsection/module-subsection';
+import { recordBoostEvent } from '$lib/utils/analytics';
+import { useMutationNotice } from '$features/ui/mutation-notice/mutation-notice';
+import styles from './speculation-method.module.scss';
+import { createInterpolateElement } from '@wordpress/element';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+
+const unsafeSpeculationRulesLink =
+	getRedirectUrl( 'jetpack-boost-speculation-rules' ) + '#unsafe_speculative_loading_conditions';
```

Comment:
We can provide hash URLs as redirects. This will allow us to control the redirect URL even when the hash anchor updates.

```suggestion
	getRedirectUrl( 'jetpack-boost-unsafe-speculation-rules' );
```

Reply:
Thanks for making the redirect!